src/ResourceManager/RedisCache/Commands.RedisCache/Models/RedisCacheAttributesWithAccessKeys.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Azure.Commands.RedisCache.Models
 
     public class RedisCacheAttributesWithAccessKeys : RedisCacheAttributes
     {
-        public RedisCacheAttributesWithAccessKeys(RedisResourceWithAccessKey cache, string resourceGroupName)
```

Comment:
@SiddharthChatrolaMs we are no longer taking breaking changes in PS. The best we can do is, add a warning for the breaking change and then in about 6 months commit the breaking change in one of the releases.
Please also help us understand the reason for this breaking change and why it cannot be address with a workaround (in this case an overload) or why it cannot wait for 6 months for the next major release of PS.